_utility-aliases.scss

```diff
@@ -1,6 +1,7 @@
 /* Border width
    ========================================================================== */
 
+$border-width--0:        $border-width--0;
```

Comment:
@paddyduke Good question! Three reasons:

* This is a numeric scale
* I think saying `0` makes the effect this property has more explicitly clear
* Our `spacing` values (which power `margin` and `padding`) are the closest we to have this pattern currently, and [they use `0` values](https://github.com/fac/fa-css-utilities/blob/master/_utility-values.scss#L132)